api/types/container/state.go

```diff
@@ -34,31 +34,3 @@ func ValidateContainerState(s ContainerState) error {
 		return errInvalidParameter{error: fmt.Errorf("invalid value for state (%s): must be one of %s", s, strings.Join(validStates, ", "))}
 	}
 }
-
-// StateStatus is used to return container wait results.
-// Implements exec.ExitCode interface.
```

Comment:
Still double-checking, but the only interface I found was `ExitCoder` in swarmkit; https://github.com/moby/moby/blob/3c8ba1529412a9ae937fe580cd1df78262e47708/vendor/github.com/moby/swarmkit/v2/agent/exec/errors.go#L39-L43

Which is matched here;
https://github.com/moby/moby/blob/master/vendor/github.com/moby/swarmkit/v2/agent/exec/controller.go#L195-L198

Which doesn't consume the response directly, but uses a `exitError` to satisfy the interface;
https://github.com/moby/moby/blob/afd6487b2e1e8be17856e7fe1eac09ac08c3d2f1/daemon/cluster/executor/container/controller.go#L316-L324